sdk/storage/Azure.Storage.Blobs/src/PartitionedUploader.cs

```diff
@@ -51,16 +53,32 @@ internal class PartitionedUploader
         public PartitionedUploader(
             BlockBlobClient client,
             StorageTransferOptions transferOptions,
-            long? singleUploadThreshold = null,
```

Comment:
The PartitionedUploaderwill now actually respect StorageTransferOptions.PartitionedDownloader